Remotion/Validation/UnitTests/Implementation/CompoundValidationMessageFactoryTest.cs

```diff
@@ -14,23 +14,160 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
+using Moq;
 using NUnit.Framework;
+using Remotion.Reflection;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.UnitTests.Implementation.TestDomain;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Implementation
 {
   [TestFixture]
   public class CompoundValidationMessageFactoryTest
   {
     [Test]
-    [Ignore("RM-5906")]
-    public void CreateValidationMessageForPropertyValidator ()
+    public void CreateValidationMessageForPropertyValidator_ContainsMessageFromSavedFactory ()
     {
+      var validatorStub = new Mock<IPropertyValidator>();
+      var propertyStub = new Mock<IPropertyInformation>();
+
+      var expectedMessage = new FakeValidationMessage("Oh no, a validation error!");
```

Comment:
Intention question: we got the InvariantValidationMessage that has the same call-site overhead in the tests. Why not just use that one instead? I do think I also went down the FakeValidationMessage route first, then had the InvariantValiationMessage and figured that's good enough and we don't need the extra weight of a fake type.

Reply:
Could you please add the reasoning / answer for this question? Thanks!

Reply:
At the point of creating the FakeValidationMessage I did not understand what the InvariantValidationMessage is used for. Therefore I simply thought going safe and creating an extra Fake to make sure that I don't misuse some functionality.